### DISCUSSION TITLE
向银保监会反映地方银行协助开发商挪用预售资金的问题 但被驳回

### DISCUSSION THREAD

**Discussion by hqvv** [WeNeedHome/SummaryOfLoanSuspension#1166] [Show and tell]
烂尾楼业主向中国银保监会反映了地方银行协助开发商挪用预售资金的问题。
但银保监会做了复议维持，把责任推给了地方政府。回复告知：因为地方政府没有制定预售资金监管办法，所以银行就可以随便乱来。  

出事了以后， 每个利益集团都在护犊子...

楼盘是 上海市崇明区长兴岛泰禾大城小院

![C949309A-AC10-4E00-BDB9-DE2F2552BE04](https://user-images.githubusercontent.com/8557480/188299260-eb214bbc-ba31-4333-a161-30a8c721344c.jpeg)

**Comment by Cegazhos** [comment]
说明银保监的都是狗杂种，和台湾死狗一样

**Reply by itsoursduty** [reply]
你的脑子是怎么做到只开化一半的

**Reply by Cegazhos** [reply]
> 你的脑子是怎么做到只开化一半的

对号入座呢🤤银保监的贱种？还是台湾死马狗？

**Comment by Sunny79928** [comment]
建议楼主研究下商品房预售资金监管制度。当地的政府相关部门（住建局）是监管主体和资金监管政策制定的主要责任人。监管银行的责任在于监管账户资金的转移和使用是否按照当地政府部门制定的政策来执行。供参考。
<img width="692" alt="image" src="https://user-images.githubusercontent.com/99377678/188416896-724455e9-7687-4277-bc25-6c626009226a.png">

**Reply by hqvv** [reply]
明明应该是人大立法的事情，十几年来 中央地方相互推诿扯皮。 导致到今天为止 上海市还没有本地的预售资金的监管政策。所以我起诉了上海市住建委。但是银监会也不能放过，等我考完试就去立案诉中国银保监会

**Comment by lettermessenger** [comment]
官官相护
